src/components/Search/CompareWithBase.tsx

```diff
@@ -95,22 +95,16 @@ function CompareWithBase({
   const [formIsDisplayed, setFormIsDisplayed] = useState(!hasEditButton);
 
   const mode = useAppSelector((state) => state.theme.mode);
+  const [searchParams] = useSearchParams();
   const styles = CompareCardsStyles(mode);
   const dropDownStyles = SearchStyles(mode);
   const hasCancelButton = hasEditButton && formIsDisplayed;
+  const frameworkURL = searchParams.get('framework');
```

Comment:
Because it's called `framework` in the query?  I'm a bit confused by this question?


```
...
  const baseRevFromUrl = url.searchParams.get('baseRev');
  const baseRepoFromUrl = url.searchParams.get('baseRepo') as
    | Repository['name']
    | null;
  const newRevsFromUrl = url.searchParams.getAll('newRev');
  const newReposFromUrl = url.searchParams.getAll(
    'newRepo',
  ) as Repository['name'][];
  const frameworkFromUrl = url.searchParams.get('framework');
...
```

Reply:
Oh you mean the name of the const. Okay, lol. Yes, I can change the name.

Reply:
I didn't mean about the name of the variable, I meant that `CompareWithBase` gets `frameworkIdVal` directly as a prop, so I don't understand why you need to get it from the search params. My understanding is that `frameworkIdVal` already has the information we need? Am I understanding this correctly?